waspc/data/Generator/templates/react-app/src/test/vitest/helpers.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
```

Comment:
You can do a named import of `ReactElement` to avoid dragging in the entire `React` namespace.